### ISSUE TITLE
Types for TypeScript.

### ISSUE THREAD

**Issue by ezequiel-umu** [Reggino/react-svg-gauge#10]
It would be great if anyone can add these types to the project.

```ts
declare module "react-svg-gauge" { 
  import { Component, CSSProperties } from "react";
 
  export interface IGaugeProps {
    label?: string;
    max?: number;
    min?: number;
    value?: number;
    width?: number;
    height?: number;
    color?: string;
    backgroundColor?: string;
    topLabelStyle?: CSSProperties;
    valueLabelStyle?: CSSProperties;
    minMaxLabelStyle?: CSSProperties;
  }

  export class Gauge extends Component<IGaugeProps> {  
    public render(): JSX.Element;
  }
  
  export default Gauge;
}
```

**Comment by Reggino** [comment]
@ezequiel-umu I'ld be happy to accept a PR!

**Comment by Reggino** [comment]
Just released v1.0.8 with this feature.
